### ISSUE TITLE
Reminder emails about class times

### ISSUE THREAD

**Issue by jzthompson** [maltyeva/mindimp-pg#112]
Could we have automated emails be sent out 24  hours in advance of each class meeting to each student in the class? The emails could be pretty basic - Something like "This is a reminder from MindImp that your English and American Literature class will meet tomorrow, May 10th, at 7:30 pm."

**Comment by maltyeva** [comment]
yes, that should definitely be possible. let me try mapping this out from a logistical perspective. So, would the classes that we have meet several times a week at the same time? Would it be something like Monday-Wednesday 10am every week or would the time change based on the day of the class? Would we want a reminder email at the beginning of the week with their schedule or one before every class? Would this be alright if a class meets 3 times a week, so the students don't start ignoring them?

**Comment by jzthompson** [comment]
Classes might mean at different times on different days (e.g. Saturday at 7:45, Wednesday at 9:45). I don't expect we'll have any classes that meet three times a week - there are certainly no plans for anything like that yet. Still, I can see your point about too many notification emails leading them to ignore everything. After all, they'll be getting notifications about assignments as well. You're right, I think just one reminder per week about the class schedule would be enough.
